cmd/check.go

```diff
@@ -99,4 +105,5 @@ func init() {
 	flags.StringSliceVar(&checkflags.AuditPlan, "audit-plan", defaultAuditPlan, "audit plan is the ordered list of operator test functions to be called during a capability audit.")
 	flags.BoolVar(&checkflags.ListPackages, "list-packages", false, "list packages in the catalog")
 	flags.StringSliceVar(&checkflags.Packages, "packages", []string{}, "a list of package(s) which limits audits and/or other flag(s) output")
+	flags.StringVar(&checkflags.LogLevel, "log-level", "", "specifies the one of the log levels in order of decreasing verbosity: debug, error, info, warn")
```

Comment:
I was suggesting this to be a root-level flag and not a check subcommand flag. But that might complicate things a bit. @bcrochet @acmenezes any thoughts?

Reply:
@yashoza19 I guess it may need to be root level and persistent to be available under any command. But I don't think it's a blocker since check is the command that most requires debugging for now. So, there is a lot of value adding it as it is. We can open an issue to optimize flags in the near future since other commands will land pretty soon.